### PR TITLE
Calendar picker

### PR DESCRIPTION
**Pull request by reed-jones**
Added new component `calendar-picker` with date selection & highlighted days

### CODE REVIEW

File: components/CalendarPicker.vue [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+        }
+    },
+    methods: {
+        generatedClass(day) {
+            let classes = [];
+            let compareDate = new Date(this.activeYear, this.activeMonth - 1, day);
+            if (this.startDate && compareDate.getTime() === this.startDate.getTime()) {
+                classes.push('cal-day-clicked')
+            } else if (this.endDate && compareDate.getTime() === this.endDate.getTime()) {
+                 classes.push('cal-day-clicked')
```
Comment (codymoorhouse) [inline components/CalendarPicker.vue]: indenting is off here

Comment (codymoorhouse) [inline components/CalendarPicker.vue]: Indenting is off here

**Review by codymoorhouse — CHANGES_REQUESTED** [review]
Can you bump the version number to 2.0.0 in package.json
Can you update the readme with the new component and information about it

**Review by codymoorhouse — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -39,5 +41,49 @@ Currently only `to` or `from` can be used separately. They cannot be used togeth
...
+    v-model='selectedDates'
+    :month.sync='month'
+    :year.sync='year' />
+```
+
+
+### Styling
+All elements are namespaced starting with `.cal` and are written at the root level for the easiest customization. The calendar will grow/shrink width in relation to the surrounding div, and the squares height can be easily overwritten using `.cal-day { height: 50px; }`. The most common styles to be overridden are likely to be the background colors of: `.cal-header, .cal-inactive, .cal-today, .cal-highlighted, .cal-clicked, .cal-faded`.
+
+![calendar-picker component](https://raw.githubusercontent.com/reed-jones/vue2-date-selector/calendar-picker/assets/calendar-picker.png)
```
Comment (kotarCreative) [inline README.md]: Should this not be pointing to the core repo?

**Review by kotarCreative — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (reed-jones) [reply README.md]: probably, but the core repo doesn't have this image until this gets merged in and i didn't want to merge a broken path by accident

Comment (kotarCreative) [reply README.md]: Well it won't be a broken path because the file and the changes go in at the same time.

**Review by kotarCreative — APPROVED** [review]

**Pull request merged**